# Topics names
lidar_topic: '/velodyne_points'
frame_id: 'velodyne'
map_topic: 'perception_map'
filtered_cloud_topic: '/filtered_points'
cones_marker_topic: 'lidar_markers'

#FOV restrictions
MAX_X_FOV: 15   # m
MAX_Y_FOV: 8   # m
MAX_Z_FOV: 0   # m
H_FOV: 180   # degrees
